new-integration-tests/src/test/resources/apps/clusterview/src/java/oracle/weblogic/kubernetes/applications/clusterview/ConfigServlet.java

```diff
@@ -178,16 +178,23 @@ private void testJdbcConnection(HttpServletRequest request, PrintWriter out) {
 
     String dsName = request.getParameter("dsName");
     String serverName = request.getParameter("serverName");
-
-    ServerRuntimeMBean serverRuntime = getServerRuntime(serverName);
-    JDBCDataSourceRuntimeMBean[] jdbcDataSourceRuntimeMBeans = serverRuntime.getJDBCServiceRuntime().getJDBCDataSourceRuntimeMBeans();
-    for (JDBCDataSourceRuntimeMBean jdbcDataSourceRuntimeMBean : jdbcDataSourceRuntimeMBeans) {
-      if (jdbcDataSourceRuntimeMBean.getName().equals(dsName)) {
-        String testPool = jdbcDataSourceRuntimeMBean.testPool();
-        if (testPool == null) {
-          out.println("Connection successful");
+    System.out.println("ITTESTS:>>>>Testing connection pool in datasource : " + dsName + " in server " + serverName);
+    try {
+      ServerRuntimeMBean serverRuntime = getServerRuntime(serverName);
+      JDBCDataSourceRuntimeMBean[] jdbcDataSourceRuntimeMBeans = serverRuntime.getJDBCServiceRuntime().getJDBCDataSourceRuntimeMBeans();
+      System.out.println("ITTESTS:>>>>Getting datasource runtime mbeans");
+      for (JDBCDataSourceRuntimeMBean jdbcDataSourceRuntimeMBean : jdbcDataSourceRuntimeMBeans) {
+        System.out.println("ITTESTS:>>>>Found JDBC datasource runtime mbean: " + jdbcDataSourceRuntimeMBean.getName());
+        if (jdbcDataSourceRuntimeMBean.getName().equals(dsName)) {
+          System.out.println("ITTESTS:>>>>Testing connection pool for JDBC datasource runtime mbean: " + jdbcDataSourceRuntimeMBean.getName());
+          String testPool = jdbcDataSourceRuntimeMBean.testPool();
+          if (testPool == null) {
+            out.println("Connection successful");
+          }
         }
       }
+    } catch (Exception ex) {
+      ex.printStackTrace();
```

Comment:
catching the exception and just printing it, does the test fail on any exceptions and how?

Reply:
sometimes when the managed servers cannot communicate with each other the test jdbc connection pool is failing in looking up the server in domain runtime tree. So I would like to get a stacktrace to debug in case test fails, the test doesn't doesn't depend on any exception to pass/fail.

Reply:
The test looks for the "Connection successful" string to pass.